resources/charlie/chainspec.toml

```diff
@@ -26,12 +26,8 @@ accounts_path = '/etc/casper/accounts.csv'
 validator_slots = 15
 
 [highway]
-# Tick unit is milliseconds.
-#
-# Timestamp for the genesis era.  At least one node has to be started when the genesis era is active in order to make
-# key blocks for the upcoming eras.  If the era is over by the time we start the nodes, they'll not be able to produce
-# blocks in it, and there won't be a new era build either.  That means when a completely new network is started, the
-# genesis era start time has to be adjusted to be active at the time.
+# Beginning of era 0. By this time, a sufficient majority (> 50% + F/2 — see finality_threshold_percent below)
+# of validator nodes must be up and running to start the blockchain.
```

Comment:
It's not necessary to _start_ but it's necessary to finalize blocks. But I understand you're making this mental shortcut or people that do not know all the internal details of the implementation.

Reply:
Also, with the current implementation, it still works if you start your node shortly after the genesis timestamp, as long as it's both within the minimum era length and the product of the minimum round length and the minimum era _height_.

But I think it's better to keep the documentation simple, even if the implementation is a bit more lenient than suggested.